tsconfig.json

```diff
@@ -5,8 +5,8 @@
 		"module": "commonjs",
 		"noErrorTruncation": true,
 		"noFallthroughCasesInSwitch": true,
-		"noUnusedLocals": true,
 		"noUnusedParameters": true,
+		"noUnusedLocals": true,
```

Comment:
NIT: these were in alphabetic order.

Reply:
This is what `jq` spits out @wrboyce:

```
$ cat tsconfig.json | jq -r --tab .
{
        "compilerOptions": {
                "allowJs": false,
                "forceConsistentCasingInFileNames": true,
                "module": "commonjs",
                "noErrorTruncation": true,
                "noFallthroughCasesInSwitch": true,
                "noUnusedParameters": true,
                "noUnusedLocals": true,
                "outDir": "build/",
                "preserveConstEnums": true,
                "removeComments": true,
                "resolveJsonModule": true,
                "sourceMap": true,
                "strict": true,
                "skipLibCheck": true,
                "strictNullChecks": true,
                "target": "es2018"
        },
        "include": [
                "src/**/*.ts",
                "typings/**/*.d.ts"
        ],
        "exclude": [
                "node_modules"
        ]
}
```

Reply:
ah, fair enough :+1:

Reply:
🙄

Reply:
there is a `-S` parameter to `jq` I just learned about today @wrboyce :)